x-pack/plugins/infra/server/lib/adapters/nodes/constants.ts

```diff
@@ -5,5 +5,11 @@
  */
 
 // TODO: Make NODE_REQUEST_PARTITION_SIZE configurable from kibana.yml
+import { InfraNodeType } from '../../../graphql/types';
```

Comment:
This separates the comment from the line it refers to. Do we want to import this above?